tlscanary/template/js/report_page.js

```diff
@@ -359,89 +344,87 @@ function apply_bootgrid()
   prop = prop.split('.');
   for (var i = 0; i < prop.length; i++) {
-      if(typeof obj[prop[i]] == 'undefined')
-          return defval;
+      if(typeof obj[prop[i]] === 'undefined')
+          return defVal;
       obj = obj[prop[i]];
   }
   return obj;
 }
 
-function transform_log(transform_data,json_data)
-{
-  var hosts = [];
-  for (var i=0;i<json_data.data.length;i++)
+function transformLog(transformData, jsonData) {
+  const hosts = [];
+  for (var i = 0;i < jsonData.data.length;i++)
   {
     var host = {};
-    for (var j=0;j<transform_data.length;j++)
+    for (var j = 0;j < transformData.length;j++)
     {
-      var prop = find_prop(json_data.data[i], transform_data[j].prop, "");
-      host[transform_data[j].name] = prop;
+      var prop = findProp(jsonData.data[i], transformData[j].prop, "");
+      host[transformData[j].name] = prop;
     }
-    hosts.push (host);
+    hosts.push(host);
   }
   return hosts;
 }
 
-function load_log(transform_data)
-{
-  var xhr = new XMLHttpRequest();
-  xhr.onload = function(arg) {
-    json_data =  JSON.parse(xhr.responseText)[0];
-    var hosts = transform_log(transform_data,json_data);
-    buildUI(json_data, hosts, transform_data);
+function loadLog(transformData) {
+  const logXHR = new XMLHttpRequest();
+  logXHR.onload = function(arg) {
+    const jsonData = JSON.parse(this.responseText)[0];
+    const hosts = transformLog(transformData,jsonData);
+    buildUI(jsonData, hosts, transformData);
   }       
-  xhr.open('GET', "log.json", true);
-  xhr.send();  
+  logXHR.onerror = function(arg) {
+    alert("Failed to load log file.")
+  }  
+  logXHR.open('GET', "log.json", true);
+  logXHR.send();  
 }
     
-function buildUI(json_data, hosts, transform_data)
-{
-  makeHeaderText(json_data.meta);
-  makeMetaTab(json_data.meta);
-  make_table (hosts, transform_data);
-  apply_bootgrid();
+function buildUI(jsonData, hosts, transformData) {
+  makeHeaderText(jsonData.meta);
+  makeMetaTab(jsonData.meta);
+  makeTable(hosts, transformData);
+  applyBootgrid();
   makeChartTab(hosts, "error");
   navigate("results");
 }
 
-function load_transform()
-{
-  var xhr = new XMLHttpRequest();
-  xhr.onload = function(arg) {
-    var transform_data =  JSON.parse(xhr.responseText);
-    columns = transform_data;
-    load_log(transform_data);
+function loadTransform() {
+  const transformXHR = new XMLHttpRequest();
+  transformXHR.onload = function(arg) {
+    const transformData =  JSON.parse(this.responseText);
+    loadLog(transformData);
+  }
+  transformXHR.onerror = function(arg) {
+    alert("Failed to load transform.json file.")
   }       
-  xhr.open('GET', "../../js/transform.json", true);
-  xhr.send(); 
+  transformXHR.open('GET', "../../js/transform.json", true);
+  transformXHR.send(); 
 }
 
-function init()
-{
-  load_transform();
+function init() {
+  loadTransform();
 }
 
-init();
\ No newline at end of file
+init();
+
+// License header
```

Comment:
Not sure what this comment line is for. Is it intentional?

Reply:
No, was a TODO comment for me that I need to remove. :)